mod/profiles/set/type/user.rb

```diff
@@ -19,31 +22,20 @@ def default_content_formgroup_args _args
         field_nest(:discussion, view: :titled, title: "Discussion", show: :comment_box),
         content_tag(:hr),
         field_nest(:activity, view: :titled, title: "Activity", hide: :menu)
-        # TODO: restore following soon
-        # field_nest(:follow, view: :profile,
-        #                    hide: [:menu, :toggle],
-        #                    title: "Following",
-        #                    items: {
-        #                      view: :content,
-        #                      structure: "User following result row"
-        #                    })
       ]
     end
   end
 
-  view :contributions_column do
-    wrap_with :div, class: "contributions-column" do
-      [wrap_with(:h4, "Contributions"), contribution_reports]
-    end
+  view :right_column do
```

Comment:
this was the problem with the profile page. I missed this view when I updated two_column_layout